src/main/java/io/confluent/idesidecar/restapi/clients/SchemaErrors.java

```diff
@@ -9,39 +9,58 @@
 import java.time.Duration;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+/**
+ * The `SchemaErrors` class manages schema error caching for different connections.
```

Comment:
Backticks don't have any meaning in JavaDoc. Normal convention is to just type the class name as normal text content (suggested), or a link such as `{@link SchemaErrors}` if you *really* want it to be aligned with the type (not really recommended here since it is the JavaDoc for that type).
```suggestion
 * The SchemaErrors class manages schema error caching for different connections.
```